bokehjs/src/coffee/tool/inspectors/hover_tool.coffee

```diff
@@ -130,6 +130,9 @@ class HoverToolView extends InspectTool.View
       vars = {index: i, x: x, y: y, vx: vx, vy: vy, sx: sx, sy: sy, data_x: data_x, data_y: data_y, rx:rx, ry:ry}
       tooltip.add(rx, ry, @_render_tooltips(ds, i, vars))
 
+      vars['renderer'] = renderer
+      @mget('callback')?.execute(vars)
```

Comment:
Sorry, reading one email at a time. Answering my own question: Yes. That's what @bryevdv is referring to.